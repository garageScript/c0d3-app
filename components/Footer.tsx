import React from 'react'

type Props = {
  footerType?: string
}

const Footer: React.FC<Props> = ({ footerType }) => {
  const footerClass = footerType
    ? `fw-light text-center ${footerType}`
    : `mt-5 fw-light text-center pb-5 text-muted`
  return (
    <footer className={footerClass}>
      &copy; Copyright {String(new Date()).split(' ')[3]} GarageScript
    </footer>
  )
}

export default Footer
