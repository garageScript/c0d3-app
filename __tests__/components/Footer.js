import React from 'react'
import { render } from '@testing-library/react'
import Footer from '../../components/Footer'

describe('Footer component', () => {
  test('Should render appropriately with footerType', () => {
    const { container } = render(<Footer footerType="py-5 bg-white text-muted" />)
    expect(container).toMatchSnapshot()
  })
})
