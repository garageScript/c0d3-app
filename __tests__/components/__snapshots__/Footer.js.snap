// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should render appropriately with footerType 1`] = `
<div>
  <footer
    class="font-weight-light text-center py-5 bg-white text-muted"
  >
    Â© Copyright 2020 GarageScript
    <br />
    All Rights Reserved
  </footer>
</div>
`;
